Domain/Entities/Flights/CargoFlight.cs

```diff
@@ -0,0 +1,15 @@
+namespace DefaultNamespace;
+
+public class CargoFlight : BaseFlight
+{
+    [Required]
+    public int CargoWeight { get; set; }
+
+    [Required]
+    public int CargoVolume { get; set; }
```

Comment:
I differentiate CargoWeight based on Weight and CargoVolume based on how much space Cargo Takes. It don't think it will precalculate anything for now it is just info for EF to make a column